crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -445,4 +445,89 @@ impl ViewportBlueprint {
             }
         }
     }
+
+    pub fn build_tree_from_containers(&self) -> egui_tiles::Tree<SpaceViewId> {
+        let mut tree = egui_tiles::Tree::empty("viewport_tree");
+
+        // Generate tile_ids for all the space-views
+        let space_view_to_tile_id: HashMap<_, _> = self
+            .space_views
+            .keys()
+            .map(|id| (id, tree.tiles.insert_pane(*id)))
+            .collect();
+
+        // Generate tile_ids for all the containers
+        let container_to_tile_id: HashMap<_, _> = self
+            .containers
+            .iter()
+            .map(|(id, container)| {
+                (
+                    id,
+                    tree.tiles
+                        .insert_container(container.to_empty_tile_container()),
+                )
+            })
+            .collect();
+
+        // Now populate children
+        for (container_id, container_blueprint) in &self.containers {
+            let Some(tile_id) = container_to_tile_id.get(container_id) else {
+                re_log::warn_once!("Missing container id that should have been generated");
```

Comment:
just use `container_to_tile_id[container_id]` instead. The ids are generated just above, we will NEVER hit this code path, yet it is eating up wasm bytes (and review time)

Reply:
Went away in the refactor anyways.